internal/server/config/config_test.go

```diff
@@ -239,7 +258,7 @@ func TestVerifyConfig(t *testing.T) {
 
 	t.Run("list_objects_deadline_request_timeout", func(t *testing.T) {
```

Comment:
Please add a similar test for ListUsers - we're missing coverage on the case where `RequestTimeout` is set alongside `ListUsersDeadline`.

RequestTimeout = <timeout> + additionalUpstreamTimeout (3s)

So if ListUsersDeadline=4s > 500ms (RequestTimeout) + 3s we expect `Verify` to return error.